docs/prover/checking/coverage-info.md

```diff
@@ -0,0 +1,107 @@
+the CVL properties. If some of the input `.sol` commands are not relevant for
+deriving the proof, it might indicate that the specification does not cover all
+behavior implemented in the smart contract. If some of the input `.spec`
+commands are irrelevant (typically unnecessary `require` statements or variable
+initializations), it indicates that the CVL rules/invariants can be made
+stronger. 
+
+You can access the coverage visualization via `Job Info -> Coverage Info page`
+buttons:
+
+![Coverage Info Button](coverage-info-button.png)
+
+
+## Examples
+
+### Tautology example
+Consider the following CVL rule called `tautology`: 
+
+```cvl
+rule tautology(uint256 fundId, method f) { 
+	env e;
+	address manager =  getCurrentManager(fundId);
+	address other;
+	require other != manager && other != e.msg.sender;
+	calldataarg args;
+	f(e,args);
+	address newManager = getCurrentManager(fundId);
+	assert newManager!= other || newManager != manager;
+}
+```
+
+Notice that we required that `other != manager` and hence the `assert` is
+necessarily `true` (see the concept of a {term}`tautology`). The executions of
+`getCurrentManager(...)` and `f(...)` are completely irrelevant. 
+
+
+The coverage visualization is shown below. It consists of two *panes*: 
+1. The left pane shows *per-line* visualization.
+2. The right pane shows detailed info about individual visualized lines. 
+
+![Example Coverage Info Visualization](tautology-sol-and-spec-cropped.png)
+
+
+In particular, in the left pane, we highlight lines from individual `.sol` and
+`.spec` files. Every line may optionally have a green, yellow, or red background
+color. No background means we have no information about this line, i.e. it might
+or might not be needed for the proof. Green, red or yellow background means we
+have some information about values of commands on the line:
+
+1. Green means that all of the values on the line are relevant for proving the
+   property. 
+2. Red means that none of the values on the line is relevant for proving the
+   property. In particular, you can arbitrary change the values and the property
+   would still hold.
+3. Yellow means that some of the values on the line are relevant and some are
+   not. 
+
+Furthermore, if we have multiple rules/invariants or a parametric rule (such as
+our `tautology`), we can also have multiple rules/invariants mapping to a single
+`.sol` or `.spec` line. In such a case, a yellow line means that some of the
+values on the line are relevant for proving some of the rules/methods/invariants
+and some of the values are not relevant. If you want to generate coverage
+visualization for a single rule or method, use {ref}`--rule` or {ref}`--method`.
+
+The right pane provides detailed information about individual values mapped to
+the lines grouped by the rule/method/invariant name (denoted *rule*). For
+instance, in the Tautology example, we can see that the value of the command
```

Comment:
either "tautology example" (lowercase) or "`tautology` example" (naming the rule).